source/_docs/locked_out.md

```diff
@@ -57,8 +57,8 @@ You will then be at the Home Assistant CLI, where you can run the custom command
 
 ### Remove the SD and access the files from another computer
 
-The files are on an EXT4 partition (`hassos-data`) and the path is `/mnt/data/supervisor`.
-These are easily accessed using another Linux machine with EXT support.
+On linux, the files are easily accessed. Modern distributions will automagically mount the partitions from the SD card.  As an
+example, in ubuntu, the files can be found in `/media/<username>//hassos-data/supervisor/homeassistant/.storage/'
```

Comment:
- `linux` -> `Linux`
- `ubuntu` -> `Ubuntu`
- Avoid double spaces after a full stop.

There is a double slash in the path? (`<username>//hassos-data`)

Please note, that this section was written towards any system. Your change made it Linux only (and aimed towards behavior specifically for Ubuntu). Can we make sure to write is inclusive towards other systems as well?

Reply:
@frenck - 
1) double slash is unintentional - but does no harm, I'll try to fix it
2) for similar improvements on Windows and OS X , I don't have access to machines running Windows or OS X. Users of those platforms should post PR's  if they wanted/needed additional clarification beyond what is currently in the section.

Reply:
> Users of those platforms should post PR's if they wanted/needed additional clarification beyond what is currently in the section.

It was written without a specific platform in mind, you've now changed it to a platform-specific text and that is ok because they should open a PR?

I think that is the wrong approach. Keep it generic and add specific details for, e.g., the Ubuntu use-case, instead of deleting the generics.

Reply:
It did not read as platform agnostic - the first path encountered in the current version is `/mnt/data/supervisor` which does not seem like a windows (would start with a drive letter...?) or an os x (would start with /Volumes?) path.  For it to be valid on linux, you should include the command that makes that path valid - something like `mount -t ext4 /dev/sdc4 /mnt/data` 
Sorry I tried to help.  Good luck with your edits.

Reply:
> first path encountered in the current version is /mnt/data/supervisor which does not seem like a windows

Correct, it is HassOS as from withing the operating system on the card itself.

> Sorry I tried to help. Good luck with your edits.

I'm doing the same, trying to help. Good luck with my edits? I'm confused, this is your PR isn't it?

Reply:
My edit is in the section 'Remove the SD and access the files from another computer' - so it's unlikely people reading this section would be in HassOS.